docs/deploy-configure.md

```diff
@@ -84,3 +122,11 @@ To upgrade this resource from `4.0.6` to `4.2.7`:
    ```
    kubectl apply -f <example>.yaml --namespace <my-namespace>
    ```
+
+## Deploy Replica Sets on OpenShift
+
+To deploy the operator on OpenShift you will have to provide the environment variable `MANAGED_SECURITY_CONTEXT` set to `true` for both the mongodb and mongodb agent containers, as well as the operator deployment.
+
+See [here](../deploy/crds/mongodb.com_v1_mongodb_openshift_cr.yaml) for an example of how to provide the required configuration for a MongoDB ReplicaSet.
```

Comment:
I think that should be "MongoDB replica set". I can't find any occurrence of "ReplicaSet" in the manual or Atlas docs.

Reply:
updated